app/controllers/user_journey_controller.rb

```diff
@@ -4,21 +4,14 @@ class UserJourneyController < ApplicationController
   include ComponentConcern
   include CertificateConcern
 
+  before_action :find_certificate
```

Comment:
```suggestion
  before_action :find_certificate except: :index
```
I don't think the `params[:certificate_id]` is present (or used) on the index page